package.json

```diff
@@ -63,6 +63,6 @@
     "@keymanapp/ldml-keyboard-constants": "file:core/include/ldml"
   },
   "engines": {
-    "node": "^18.x"
+    "node": "20.16.0"
```

Comment:
Should we publish this in a manner that external developers utilize one or more of our TS/JS libraries, this will generate "bad engine" warnings if _their_ repository's version of node does not match what we've set here.

That is to say, we're using a "big hammer" here that's not the intended purpose for this field of package.json.  It _is_ a convenient place for us to reference, I'll admit, but this could be problematic for others.

Any reason we couldn't just make a simple CI_NODE_VERSION.md file and reference that instead, leaving the package's entry here the range for better external compatibility?

Reply:
I understand what you are saying. One question: is anyone likely to be using our repo in this way -- as opposed to using our published packages?

I could use the minimum_versions.inc.sh value instead, and make it a specific version for node.

Reply:
Right... using our packages doesn't pull in the _repo base_ package.json.  Forgot about that during the first pass here.

Yeah, it's probably fine then.  As long as we don't copy this specificity to the child packages, I think we're good.

Reply:
> Any reason we couldn't just make a simple CI_NODE_VERSION.md file

We basically already have that with `docs/minimum-versions.md`.

Reply:
Putting the specific version here allows other tooling such as NPM itself to complain if there are version mismatches. For example, we see that with eslint.